pancancer-arch-3/src/main/java/info/pancancer/arch3/worker/WorkerRunnable.java

```diff
@@ -188,7 +192,9 @@ public void run() {
                             workflowResult.setWorkflowStdout("everything is awesome");
                             workflowResult.setExitCode(0);
                         } else {
-                            workflowResult = launchJob(statusJSON, job);
+                            String seqwareEngine = settings.getString(Constants.WORKER_SEQWARE_ENGINE);
```

Comment:
FYI, for the future. You can actually specify the default as the second parameter to getString and its cousins. 
Please don't specify it in the code later (as in this) because it makes the default harder to track-down. Thanks!